flink-runtime/src/main/java/org/apache/flink/runtime/io/network/api/serialization/SpanningRecordSerializer.java

```diff
@@ -151,6 +176,15 @@ private SerializationResult getSerializationResult() {
 		return SerializationResult.PARTIAL_RECORD_MEMORY_SEGMENT_FULL;
 	}
 
+	/**
+	 * Gets the currently set target buffer and sets its size to the actual
+	 * number of written bytes.
+	 *
+	 * After calling this method, a new target buffer is required to continue
+	 * writing (see {@link #setNextBuffer(Buffer)}).
+	 *
+	 * @return the target buffer that was used
+	 */
```

Comment:
Shouldn't these javadocs be added to the RecordSerializer interface instead? (the same applies to other implemented methods)